_includes/scenarios/integrate/propose-attribute-to-peer.md

```diff
@@ -1,5 +1,215 @@
+          }
+        },
+        {
+          "@type": "ProposeAttributeRequestItem",
+          "mustBeAccepted": false,
+          "attribute": {
+            "@type": "IdentityAttribute",
+            "owner": "",
+            "value": {
+              "@type": "PhoneNumber",
+              "value": "<phone number that the Sender proposes to the Recipient>"
+            }
+          },
+          "query": {
+            "@type": "IdentityAttributeQuery",
+            "valueType": "PhoneNumber"
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+In our example, the Sender only requires the Recipient to accept the ProposeAttributeRequestItems belonging to the PersonName and the EMailAddress, which is why the individual [ProposeAttributeRequestItems]({% link _docs_integrate/data-model-overview.md %}#proposeattributerequestitem) and the [RequestItemGroup]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup) within the Request have specified corresponding values in their `mustBeAccepted` property. We assume that the Recipient wants to accept the Request and all its ProposeAttributeRequestItems with the exception of the PhoneNumber.
+
+If the Recipient wants to accept the Request for proposing Attributes, it must accept all ProposeAttributeRequestItems for which the `mustBeAccepted` property is set to `true`. It is therefore not permitted for the Recipient to refuse to accept the ProposeAttributeRequestItem belonging to the PersonName or the EMailAddress.
+{: .notice--info}
+
+We assume that the Recipient confirms the validness of the PersonName proposed by the Sender and that the Sender has proposed an outdated EMailAddress to the Recipient for creation. The Recipient therefore wants to create a corrected version of the EMailAddress and send it back to the Sender. The Recipient accepts at least one ProposeAttributeRequestItem of the RequestItemGroup, rejects the PhoneNumber and accepts the ProposeAttributeRequestItems belonging to the PersonName and the EMailAddress. Consequently, it responds to the Request for proposing Attributes as follows:
+
+```jsonc
+{
+  "items": [
+    {
+      //Accept the creation of the proposed PersonName
+      "accept": true,
+      "attribute": {
+        "@type": "IdentityAttribute",
+        "owner": "",
+        "value": {
+          "@type": "PersonName",
+          "givenName": "<given name that the Sender proposes to the Recipient>",
+          "surname": "<surname that the Sender proposes to the Recipient>"
+        }
+      }
+    },
+    {
+      //Accept RequestItemGroup
+      "accept": true,
+      "items": [
+        {
+          //Create a corrected version of the proposed EMailAddress
+          "accept": true,
+          "attribute": {
+            "@type": "IdentityAttribute",
+            "owner": "",
+            "value": {
+              "@type": "EMailAddress",
+              "value": "<corrected version of the email address proposed by the Sender>"
```

Comment:
`"<version of the email address corrected by the Recipient>"` would maybe be a bit clearer

Reply:
`<Recipient's corrected version of the email address>`